src/components/HungerAlert/style.css

```diff
@@ -0,0 +1,44 @@
+.pulse {
```

Comment:
as already mentioned we do not use css and use tailwind instead

Reply:
yeah I wish I can animate and have pseudo-elements in tailwind as well but I didnt find any.